media-fonts/lxgw-wenkai/metadata.xml

```diff
@@ -5,10 +5,15 @@
 		<email>rtgiskard@gmail.com</email>
 		<name>Zheng Zhu</name>
 	</maintainer>
+	<maintainer type="person">
+		<email>xen0n@gentoo.org</email>
+		<name>WANG Xuerui</name>
+	</maintainer>
 	<maintainer type="project" proxied="proxy">
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
+	<stabilize-allarches/>
```

Comment:
(I wonder about this, do we have previous examples or policies for this?)

Reply:
I think packages that are pure data (i.e. architecture-agnostic) should all be marked `<stabilize-allarches/>`, obviously; I remember reading about this somewhere but definitely not in the devmanual nor quiz. Maybe just in nattka docs and tree.

Reply:
It is in devmanual, 
https://devmanual.gentoo.org/keywording/#simultaneous-stabilization-on-all-architectures

I just personally don't know if it applies to font files. If I think about it, it does, but then again it executes fontconfig so I can also imagine it can work differently on different architechtures. (Also the weird interaction with X use flag...?)

Anyway I'd do ack putting font files as allarches, I just haven't seen any examples of that being done and wonder if there's a reason for that.

Reply:
Hmm surely I've already forgotten about the details! (I just grepped my quiz and there was no results, assuming almost everything in devmanual is being tested.)

We could definitely use some mass-marking though... maybe on architectures without fontconfig we could simply skip refreshing the font cache, it's not necessary in every possible scenario after all. While I can't think of any reason not to use fontconfig for, well, *configuring* fonts (font matching is basically a big pile of declarative configurations), font data is certainly usable without involving fontconfig.